server/controllers/finance/reports/generalLedger/index.js

```diff
@@ -35,20 +35,22 @@ function renderReport(req, res, next) {
   });
   let report;
   let data;
-  const currentDate = new Date();
 
   try {
     report = new ReportManager(REPORT_TEMPLATE, req.session, options);
   } catch (e) {
     return next(e);
   }
 
-  return Fiscal.getPeriodCurrent(currentDate)
+  let fiscal_year_id = options.fiscal_year_id;
```

Comment:
I don't think the staff at the hospital would notice either way - it would be good to make a decision as BHIMA though. 

We will see a similar behavior I think on the account statement module selecting a single account to drive the module. 

My opinion is that the modal just adds a lot of boilerplate that we wouldn't otherwise need but I don't think it's a bad user experience.